RecoEgamma/EgammaPhotonProducers/python/gedPhotons_cfi.py

```diff
@@ -94,7 +98,50 @@
     RecHitSeverityToBeExcludedEB = cleanedHybridSuperClusters.RecHitSeverityToBeExcluded,
     RecHitFlagToBeExcludedEE = multi5x5BasicClustersCleaned.RecHitFlagToBeExcluded,
     RecHitSeverityToBeExcludedEE = cleanedHybridSuperClusters.RecHitSeverityToBeExcluded,
-    checkHcalStatus = cms.bool(True)
+    checkHcalStatus = cms.bool(True),
+    PhotonDNNPFid = cms.PSet(
+        enabled = cms.bool(False),
```

Comment:
was this meant to be enabled ? ref #42949

Reply:
its enabled via era, as it is meant to work only for run 3.

https://github.com/cms-sw/cmssw/blob/aca7593194f3f57ea0b3ac0c82100d11b098d708/RecoEgamma/EgammaPhotonProducers/python/gedPhotonSequence_cff.py#L58-L62